crates/egui-winit/src/lib.rs

```diff
@@ -836,26 +836,25 @@ impl State {
         }
 
         if let Some(ime) = ime {
-            let pixels_per_point = pixels_per_point(&self.egui_ctx, window);
-            let ime_rect_px = pixels_per_point * ime.rect;
-            if self.ime_rect_px != Some(ime_rect_px)
-                || self.egui_ctx.input(|i| !i.events.is_empty())
-            {
-                self.ime_rect_px = Some(ime_rect_px);
+            let ime_rect = if ime.visible {
+                ime.cursor_rect
+            } else {
+                self.egui_ctx.screen_rect()
```

Comment:
It's still visible though, right? Just applied to the whole screen instead of just the text cursor?

Reply:
IME is not displayed within the specified area. In other words, the IME is not displayed within the program and is not visible.
(I'm not sure if this is only on Windows, but my current method is the best.)

In the end, we can control it according to own needs by selecting visibility, etc. depending on the situation.